src/lib/holocene/select/multi-select.svelte

```diff
@@ -27,9 +27,15 @@
   export let selectAllLabel: string;
   export let clearAllLabel: string;
   export let active = false;
+  export let disabled = false;
   export let position: 'left' | 'right' = 'left';
+  export let initialSelectedAll = true;
```

Comment:
Follow-up: I'm not actually sure if this is the default we want. IMO a multi-select should start with none selected. Also, if we already have an `initialSelected` prop, can't the Cloud page just pass that prop in as the whole list of namespaces?

Reply:
It's a bit tricky, because all our other multi-selects we want all selected because they are filters (include all filters...then remove them).

I'll follow up with something better here for sure.